src/engine.cc

```diff
@@ -107,6 +107,7 @@ SearchLimits EngineController::PopulateSearchLimits(int /*ply*/, bool is_black,
   // reduce it.
   if (slowmover < 1.0 || this_move_time > kSmartPruningToleranceMs) {
     // Budget X*slowmover for current move, X*1.0 for the rest.
+    slowmover *= pow( (1 + exp((ply - 64.5) / 6.85)), -0.171) + DBL_MIN
```

Comment:
std::pow
std::exp
it's better to add `f` suffix after all constants, hoping that everything is float instead of double, e.g. `64.5f`.
`slowmover` is float, so DBL_MIN should be FLOAT_MIN, or better std::numeric_limits<float>::min()

Reply:
...but actually that PR doesn't do what it should... It seems that it will work in opposite way.

During first moves, it doesn't "see" that later it will need less.. Hmm, need to think more about it, but i'm find with submitting this variant to start with.